examples/use_rust/py32f07x/Cargo.toml

```diff
@@ -0,0 +1,65 @@
+[package]
+name = "rmk-py32f07x"
+version = "0.2.0"
+authors = ["Haobo Gu <haobogu@outlook.com>"]
+description = "Keyboard firmware written in Rust"
+homepage = "https://github.com/haobogu/rmk"
+repository = "https://github.com/haobogu/rmk"
+readme = "../../README.md"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+
+[dependencies]
+rmk = { path = "../../../rmk", features = ["_no_external_storage"] }
+embassy-time = { version = "0.3", features = ["defmt"] }
+py32-hal = { version = "0.2.0", features = [
+# py32-hal = { git = "https://github.com/py32-rs/py32-hal", rev = "ecfec8f", features = [
+    "py32f072c1b", 
+    "memory-x", 
+    "embassy", 
+    "rt", 
+    "time-driver-tim3"
+] }
+
+
+embassy-executor = { version = "0.6.0", features = [
+    "defmt",
+    "arch-cortex-m",
+    "executor-thread",
+    "integrated-timers",
+    "task-arena-size-32768",
```

Comment:
py32f072 has only 16K RAM, so setting task arena size to 32768 might not work.

Reply:
Fixed. Thanks!